src/errexit.f

```diff
@@ -1,23 +1,20 @@
 C> @file
-C
-C> EXIT WITH A RETURN CODE
-C> @author IREDELL @date 1998-06-04
+C> @brief Exit with a return code.
+C> @author Mark Iredell @date 1998-06-04
 C>
-C> EXIT WITH A RETURN CODE
+C> Exit with a return code.
 C>
-C> PROGRAM HISTORY LOG:
-C>   1998-06-04  IREDELL
-C>   1999-01-26  Gilbert       changed to use XLF utility routine exit_(n)
-C>                             instead of exit(n).  exit_(n) will return
-C>                             the proper value ( n must be 4 byte int )
-C>                             to the sh/ksh shell status variable $?
-C>                             ( $status for csh ) on the IBM SP.
+C> Program history log:
+C> - 1998-06-04  Mark Iredell
+C> - 1999-01-26  Stephen Gilbert
+C>  - Changed to use XLF utility routine exit_(n) instead of exit(n).
+C>    exit_(n) will return the proper value (n must be 4 byte int)
+C>    to the sh/ksh shell status variable $? ($status for csh)
+C>    on the IBM SP.
 C>
-C> @param[in] IRET INTEGER RETURN CODE
-C>
-C> SUBPROGRAMS CALLED:
-C>   EXIT      - EXITS FROM A FORTRAN PROGRAM
+C> @param[in] IRET Integer return code.
```

Comment:
Surely this is not an in param, but an out param.

Reply:
It's weird, if you look at the code it seems to me to be an in param (but also a weird way for an out param). I will move it to an out param if you disagree still.

Reply:
OK, stick with what the code says.

There will be further opportunity to tune the documentation as we write tests. For now, we want to get it all pretty decent, and clear all doxygen warnings, so we can turn on doxygen checking for future PRs. The next step will be to start writing tests, and that will involve updating the documentation some more as we learn about the code.

Reply:
also noting that the original documentation has it as an input param